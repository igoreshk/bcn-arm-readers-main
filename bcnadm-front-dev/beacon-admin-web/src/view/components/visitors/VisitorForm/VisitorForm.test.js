import React from 'react';
import { shallow } from 'enzyme';

import { VisitorForm } from './index';

const props = {
  currentVisitor: {},
  handleSubmit: jest.fn(),
  history: {},
  translate: jest.fn(),
  valid: true
};

const wrapper = shallow(<VisitorForm {...props} />);

describe('VisitorForm', () => {
  it('Check matching snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('Check component has rendered', () => {
    expect(wrapper.length).toEqual(1);
  });
});
