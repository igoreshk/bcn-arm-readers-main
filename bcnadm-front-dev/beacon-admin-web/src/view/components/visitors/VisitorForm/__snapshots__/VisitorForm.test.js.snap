// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VisitorForm Check matching snapshot 1`] = `
<WithStyles(ForwardRef(Dialog))
  open={true}
>
  <WithStyles(ForwardRef(DialogTitle)) />
  <WithStyles(ForwardRef(DialogContent))>
    <form
      className="visitorForm"
    >
      <div
        className="firstBlock"
      >
        <div>
          <Field
            component={[Function]}
            name="name"
          />
        </div>
      </div>
      <div
        className="secondBlock"
      >
        <Field
          component={[Function]}
          name="deviceId"
        />
      </div>
    </form>
  </WithStyles(ForwardRef(DialogContent))>
  <WithStyles(ForwardRef(DialogActions))>
    <WithStyles(ForwardRef(Button)) />
    <WithStyles(ForwardRef(Button))
      color="primary"
      disabled={true}
    />
  </WithStyles(ForwardRef(DialogActions))>
</WithStyles(ForwardRef(Dialog))>
`;
